firmware/src/apps/light_dimmer/light_dimmer.cpp

```diff
@@ -470,7 +475,14 @@ TFT_eSprite *LightDimmerApp::render()
             adjusted_angle = right_bound;
         }
         spr_->fillSmoothCircle(TFT_WIDTH / 2 + (screen_radius - 10) * cosf(adjusted_angle), TFT_HEIGHT / 2 - (screen_radius - 10) * sinf(adjusted_angle), 5, dot_color, foreground_color);
+        led_ring_notifier->staticColorRange((255 << 16) | (127 << 8) | 0, 255, wanted_angle * 180 / PI, start_angle * 180 / PI + 5, LED_RING_PRIOTITY_APP);
```

Comment:
can we add a bit of comments to this line (ie.: what is 5 in the start_angle * 180 / PI + 5) so for posterity we don't waste to much time tracing it back